tests/test-dfa.cpp

```diff
@@ -178,6 +139,33 @@ TEST_CASE("Test Complex Tagged DFA", "[DFA]") {
             "7:accepting_tags={0},accepting_operations={8c16,9c19,10c20,11c21,12c17,13c22,14c27,"
             "15c28},byte_transitions={}\n"
     };
+    test_dfa({var_schema}, expected_serialized_dfa);
+}
+
+TEST_CASE("Test Repetition Tagged DFA", "[DFA]") {
```

Comment:
nit: for consistency with other tests
```suggestion
TEST_CASE("Test repetition tagged DFA", "[DFA]") {
```

Reply:
Is "repeating tagged DFA" or "tagged repetition DFA" better? The current one reads a bit weird to me.